_includes/doc/admin-guide/options/persist-name.md

```diff
@@ -9,6 +9,10 @@ drivers:
 
 > Error checking the uniqueness of the persist names, please override it with persist-name option. Shutting down.
 
+or
+
+> Automatic assignment of persist names failed, as conflicting persist-names were found. Please override the automatically assigned identifier using an explicit perist-name() option or remove the duplicated configuration elements
```

Comment:
Perhaps a unified naming would be better, use "persist names" or "persist-names" everywhere. Also, a typo in "perist-name()" and a missing dot. 

```suggestion
> Automatic assignment of persist names failed, as conflicting persist names were found. Please override the automatically assigned identifier using an explicit persist-name() option or remove the duplicated configuration elements.
```